docs/next.config.js

```diff
@@ -243,4 +243,13 @@ module.exports = withDocsInfra({
       { source: `/static/x/:rest*`, destination: 'http://0.0.0.0:3001/static/x/:rest*' },
     ];
   },
+  redirects: async () => {
```

Comment:
We keep the redirects in `docs/public/_redirects`, please remove this and move the redirect entry there.